lib/filter_nb.c

```diff
@@ -574,10 +564,10 @@ static int lib_access_list_entry_destination_network_destroy(
 
 	f = nb_running_get_entry(args->dnode, NULL, true);
 	fc = &f->u.cfilter;
-	fc->extended = 0;
-	cisco_unset_addr_mask(&fc->mask, &fc->mask_mask);
+	fc->extended = 1;
+	yang_dnode_get_ipv4(&fc->mask_mask, args->dnode, NULL);
 
-	acl_notify_route_map(f->acl, RMAP_EVENT_FILTER_DELETED);
+	acl_notify_route_map(f->acl, RMAP_EVENT_FILTER_ADDED);
```

Comment:
Should we need to notify when we process prefix and mask separately?

Reply:
Yes, otherwise if you just change one of these it will cause the entry not getting updated.

Reply:
Since this wildcard is option this may be needed but it may lead to performace issue in VTY we used to process all address and wildcard at once but here it is split in to two.

Reply:
Thinking over it again we may not be able to do anything here as it is an option and may not be present and address can be present on its own.

dev(config)# access-list 20 seq 1 permit 10.10.10.0 
  <cr>     
  A.B.C.D  Wildcard bits

Reply:
This is a performance optimization, I'd leave it for another PR. We could move all the `acl_notify_route_map` code to a single `apply_finish` (see https://github.com/FRRouting/frr/blob/master/lib/northbound.h#L387), but it'd require changing code outside the scope of this PR.

I also don't know exactly how that callback works, but it could also open a possibility for us having no room to call the `RMAP_EVENT_FILTER_DELETED` version then having to come up with another thing to fix it (`apply_finish` does not cover `destroy` case).